arrange_schedule.py

```diff
@@ -1222,21 +1222,16 @@ def crawler_ptt_news(boards):
     try:
         return_msg = {}
         return_msg["result"] = "fail"
-        data_type = 7
```

Comment:
I don't understand `data_type` doing. Why is this variable needed in the beginning?